packages/master/master.js

```diff
@@ -89,12 +89,14 @@ const wsActiveConnectionsGauge = new libPrometheus.Gauge(
 	"How many WebSocket connections are currently open to the master server",
 	{ callback: function(gauge) { gauge.set(master.wsServer.activeConnectors.size); }}
 );
+void wsActiveConnectionsGauge;
```

Comment:
Also don't understand the point of this. `void [object]` simply evaluates the object to an object and returns `undefined`. This looks like a noop?

Reply:
Same with this. `void wsActiveConnectionsGauge;` doesn't do anything but looks like a usage of the variable for static analysis tools.

Reply:
If the variable really is unused, why not just do `void new libPrometheus.Gauge(...)`?

Reply:
That is an option but seems a bit weird to create anonymous metrics to me.

Reply:
I mean, it is though. Putting it in a variable indicates that the variable is passed to be used somewhere. In this case, the variable isn't used and can't really be used since  the metric is updated through the callback.

Reply:
It can be used to retrieve the value, e.g. wsActiveConnectionsGauge.get() for a status report. But the variable can be added when that status report is added.